.github/workflows/builds.yml

```diff
@@ -0,0 +1,119 @@
+name: Dev Releases
+
+on:
+  push:
+    branches:
+      - dev
+
+env:
+  LANG: en_US.UTF-8
+  ENVFILE: .env.dev
+  # App runtime environment
+  API_URL: ${{ secrets.DEV_API_URL }}
+  # S3
+  FOLDER: react-native-base
+  AWS_REGION: us-east-1
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  KEYS_BUCKET: ${{ secrets.AWS_S3_KEYS_BUCKET }}
+  BUILDS_BUCKET: ${{ secrets.AWS_S3_BUILDS_BUCKET }}
+  # iOS Release
+  APPLE_PROFILE: ${{ secrets.APPLE_DEV_PROFILE }}
+  APPLE_CERT: ios_distribution.cer
+  APPLE_KEY: Certificates.p12
+  APPLE_KEY_PASSWORD: ${{ secrets.APPLE_KEY_PASSWORD }}  
+  APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID}}
+  FASTLANE_ITC_TEAM_ID: ${{ secrets.APPLE_TEAM_ITC_ID}}
+  FASTLANE_USER: ${{ secrets.FASTLANE_USER }}
+  FASTLANE_PASSWORD: ${{ secrets.FASTLANE_PASSWORD }}  
+  FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: ${{ secrets.FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD }}
```

Comment:
The APP_STORE_CONNECT_API_KEY_FILE,
 APP_STORE_CONNECT_API_KEY_KEY_ID and 
 APP_STORE_CONNECT_API_KEY_ISSUER_ID 
 are missing